visualization/index.html

```diff
@@ -135,141 +143,130 @@ <h1>
 							<br/><br/>
-							The vertical axis, that is, the height of the point on the graph, represents the average number of daily visits during the year analyzed. Dashed lines indicate average daily visits in the previous year.
+							<span data-i18n="how-to-paragraph-sentence3">The vertical axis, that is, the height of the point on the graph, represents the average number of daily visits during the year analyzed. Dashed lines indicate average daily visits in the previous year.</span>
 							<br/><br/>
-							In the menu above you can choose a subject and the order of the articles on the horizontal axis. Hover over the circle to see more information, and click to go to the Wikipedia article.
+							<span data-i18n="how-to-paragraph-sentence4">In the menu above you can choose a subject and the order of the articles on the horizontal axis. Hover over the circle to see more information, and click to go to the Wikipedia article.</span>
 						</p>
+						
 					</div>
 					<div class="column_3">
 						<div id="dv1_legend_a" class="legend svg-element">
-							<img src="assets/img/dv1_legend_a.svg" width="326" height="262" alt="The light-colored circle represents the size of the article, the dark-colored circle represents the introduction dimension, and the circumference represents the size of the talk page.">
+							<img id="img_a" src="assets/img/dv1_legend_a_es.svg" width="326" height="262" alt="The light-colored circle represents the size of the article, the dark-colored circle represents the introduction dimension, and the circumference represents the size of the talk page.">
 						</div>
 						<div id="dv1_legend_b" class="legend svg-element">
-							<img src="assets/img/dv1_legend_b.svg" width="326" height="155" alt="The pink color represents the social sciences and humanities, the green color represents the scientific-mathematical area, the light blue color represents communication and the arts and the violet color represents the technical-technological area.">
+							<img id="img_b" src="assets/img/dv1_legend_b_es.svg" width="326" height="155" alt="The pink color represents the social sciences and humanities, the green color represents the scientific-mathematical area, the light blue color represents communication and the arts and the violet color represents the technical-technological area.">
 						</div>
-					</div>
+					</div>					
 					<div class="column_3">
-						<h2>Data</h2>
+						<h2 data-i18n="data">Datos</h2>
 						<p>
-							The third cycle school curriculum is <a href="https://www.wikidata.org/wiki/Wikidata:Wikidata_for_Education" target="_blank" aria-label="modelada en Wikidata">modeled on Wikidata</a>.
```

Comment:
Hello @Bhbee!
It looks very good! Almost ready to merge... but please first fix this line (we missed the "modelled on wikidata" part)

I have some other less comments, let's discuss tomorrow at the call. See you!

Reply:
oh, I see it!

I've corrected that.